apps/backend/src/modules/notifications/notification.controller.ts

```diff
@@ -53,7 +59,22 @@ class NotificationController extends BaseController {
 	 * @swagger
 	 * /notifications:
 	 *   get:
-	 *     description: Returns an array of notifications
+	 *     description: Returns an array of notifications with pagination
+	 *     parameters:
+	 *       - in: query
+	 *         name: page
+	 *         schema:
+	 *           type: integer
+	 *           minimum: 1
+	 *           default: 1
+	 *         description: The page number to retrieve
+	 *       - in: query
+	 *         name: pageSize
+	 *         schema:
+	 *           type: integer
+	 *           minimum: 1
+	 *           default: 10
```

Comment:
Here you mentioned default values for page and pageSize, but there is no code for this ( frontend doesn't counted :0 )
So what would happen if we try to send get request without any query params?

Reply:
I updated the Swagger documentation to match the backend code. Initially, I had set default values for page and `pageSize `in the backend, but I realized that the frontend already manages those values in the `NotificationsPopover` component and `useInfiniteScroll `hook. So, there's no need for default handling in the backend.